### ISSUE TITLE
False positive: geolocation.onetrust.com breaks logging into slate.com

### ISSUE THREAD

**Issue by aclight** [nextdns/metadata#541]
In the native privacy blocklist, the rule blocking geolocation.onetrust.com prevents logging in and managing user profiles at slate.com. An example URL: https://slate.com/sign-in

The blocklist file from which this comes (https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts) also has a comment about how this rule affects cnet.com, though I haven't tested that site.

**Comment by crssi** [comment]
Stop using "native" privacy blocklist... it has too many such troubles.

Use instead: 1hosts (lite), anudeep and oisd... and that is all you need and nothing more.
Additionally, just add `graph.instagram.com` to `Allowlist` if users use FB or Insta.

**Comment by aclight** [comment]
If I recall correctly, the "native" blocklist is enabled by default. At least if I create a new configuration, it comes with that blocklist selected. So even if I stop using it, new users will be impacted by this.

I have allowlisted geolocation.onetrust.com for my own configuration. I saw that in #505 a different geolocation server was handled specially in the native blocklist for the same reason, so I thought I would report it here.
